cmd.go

```diff
@@ -0,0 +1,85 @@
+package wish
+
+import (
+	"fmt"
+	"io"
+	"os/exec"
+	"runtime"
+	"time"
+
+	"github.com/charmbracelet/ssh"
+)
+
+// Command sets stdin, stdout, and stderr to the current session's PTY slave.
+//
+// If the current session does not have a PTY, it sets them to the session
+// itself.
+func Command(s ssh.Session, name string, args ...string) *Cmd {
+	c := exec.Command(name, args...)
+	pty, _, ok := s.Pty()
+	if !ok {
+		c.Stdin, c.Stdout, c.Stderr = s, s, s
+		return &Cmd{cmd: c}
+	}
+
+	return &Cmd{c, &pty}
+}
+
```

Comment:
```suggestion
	return command(s, c)
}

func CommandContext(ctx context.Context, s ssh.Session, name string, args ...string) *Cmd {
	c := exec.CommandContext(ctx, name, args...)
	return command(s, c)
}

func command(s ssh.Session, c *exec.Cmd) *Cmd {
	pty, _, ok := s.Pty()
	if !ok {
		c.Stdin, c.Stdout, c.Stderr = s, s, s
		return &Cmd{cmd: c}
	}

	return &Cmd{c, &pty}
}

```

Add in a `CommandContext` wrapper as well.